test/08_manylinuxXXXX_only/spam.c

```diff
@@ -12,7 +18,9 @@ spam_system(PyObject *self, PyObject *args)
     if (!PyArg_ParseTuple(args, "s", &command))
         return NULL;
 
-#if defined(__linux__)
+#if defined(__GLIBC_PREREQ) && __GLIBC_PREREQ(2, 17)  /* manylinux2014 is glibc 2.17 */
+    sts = (int)secure_getenv("NON_EXISTING_ENV_VARIABLE");
```

Comment:
```suggestion
    // secure_getenv is only available in manylinux2014
    sts = (int)secure_getenv("NON_EXISTING_ENV_VARIABLE");
```

Reply:
To be pedantic:

"ensuring a manylinux2014-only wheel is produced", or "ensuring a manylinux2014 wheel is produced that is not manylinux2010-compatible"

Reply:
good point. I've edited the above